### DISCUSSION TITLE
Moving form submitted to collection

### DISCUSSION THREAD

**Discussion by idaksh6** [statamic/cms#7723] [Q&A] [answered]
Hi. I have one problem. I built website via statamic. I made a form to submit the review but now customer wants that if form is submitted it has to go to collection and if they approved this review it needs to display in website. How to achieve this? Thank you

**Comment by duncanmcclean** [comment, marked as answer]
Hey! You probably have two options to achieve this:

1. Build an 'Approve' action for your form submissions. When they're approved, create a new entry using the form submission data.
2. Use the [Guest Entries](https://statamic.com/addons/duncanmcclean/guest-entries) addon so any form submissions on the front-end are saved as entries. All you'd need to do to make them live is to mark them as published.

_Disclaimer:_ I'm the developer of the Guest Entries addon.
